network-api/networkapi/mozfest/templates/partials/signup.html

```diff
@@ -1,4 +1,9 @@
+{% load i18n %}
+
 <div class="cta col-lg-4 offset-lg-1">
+  <div class="narrow-sticky-button-container hidden">
+    <a href="#cta-anchor" class="btn btn-primary d-block">{% trans "SIGN UP" context "Sticky button in mobile view on MozFest homepage" %}</a>
```

Comment:
is this a real href?

Reply:
Oh yea we want the button to scroll to the `<div id="cta-anchor">` node when user clicks it.